success-criteria.md

```diff
@@ -337,21 +309,23 @@ stakeholder group.
 
         -   Free elections protected against foreign manipulation
 
-            -   While free speech requires allowing speech the majority does not
-                approve of, we can label political speech by the author’s
-                nationality and whether it is endorsed by one or more candidates
-
-        -   Freedom of the press enables watch-dog reporting on important issues
-            and combats fake news
-
     -   Freedom of information to provide fast, easy access to internet-enabled
         content for all
 
         -   Cost-free access enables access for all, regardless of economic
             means
 
-        -   Freedom from having to self-censor for fear of content consumption
-            being associated with directly-identifiable, offline identity
+-   Freedom of the press, which enables and supports:
```

Comment:
Freedom of the press should be a major point under the interests of society as a stakeholder group. It also has its own societal level concerns that we should account for relative to this document.